airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/test/check/CheckIntegrationTest.kt

```diff
@@ -76,7 +89,7 @@ open class CheckIntegrationTest<T : ConfigurationJsonObjectBase>(
                 { assertEquals(AirbyteConnectionStatus.Status.FAILED, connectionStatus.status) },
                 {
                     assertTrue(
-                        failurePattern.matcher(connectionStatus.message).matches(),
+                        failurePattern.matcher(connectionStatus.message).find(),
```

Comment:
me being unfamiliar with java's regex tooling: `matches` requires the entire message to match the regex; `find` will return true if any substring matches the regex